lib/mayaUsd/ufe/ProxyShapeHierarchy.cpp

```diff
@@ -34,6 +34,8 @@
 #include <mayaUsd/ufe/UsdUndoReorderCommand.h>
 #endif
 
+PXR_NAMESPACE_USING_DIRECTIVE
```

Comment:
Simpler and more readable to keep the using directive in the implementation, where the number of changes is high.